CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,78 @@
+# Ilios Code of Conduct
+
+Like the technical community as a whole, the Ilios team and community is made up
+of a mixture of professionals and volunteers from all over the world, working on
+every aspect of the mission - including mentorship, teaching, and connecting
+people.
+
+Diversity is one of our huge strengths, but it can also lead to communication
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors and those
```

Comment:
I would change `This code applies equally to founders, mentors and those seeking help and guidance.` to 

`this code applies equally to contributors,  mentors, and those seeking help and guidance`